trulens_eval/trulens_eval/feedback/v2/feedback.py

```diff
@@ -305,6 +303,8 @@ class PromptResponseRelevance(Relevance, WithPrompt):
 
         - RESPONSE that is only seemingly RELEVANT should get a score of 0.
 
+        - Answers that intentionally do not answer the question, such as 'I don't know' and model refusals, should also be counted as the least RELEVANT and get a score of 0.
+
```

Comment:
Does this work well or should it be like:
Answers that intentionally do not answer the question, such as 'I don't know' and model refusals, should also be counted as the least RELEVANT and get a score of 0, when all information to answer is part of PROMPT.

Reply:
Answerability is not considered here, only in groundedness